### DISCUSSION TITLE
Having issue reporting a bug?

### DISCUSSION THREAD

**Discussion by elad2109** [flameshot-org/flameshot#2559] [Q&A]
I've tried to report but the "submit" button is disabled. Filled all mandatory fields.
By the same token, here is also the actual report I want to report.

Flameshot v11.0.0 (11.0.0-2 Debian)
Compiled with Qt 5.15.2
linux: 5.15.15-1rodete2-amd64
debian: rodete

1. run flameshot ui
2. click on the measurement tool.
3. Nothing happen. The icon is not even colored selected.

https://drive.google.com/file/d/1-Umhb1FDtsOpqX8D4n2hUqlE92EibqNQ/view?usp=sharing
![image](https://user-images.githubusercontent.com/1093841/164949027-a67c3e7a-f4be-4192-a741-4b34ec1d9408.png)

**Comment by mmahmoudian** [comment]
> I've tried to report but the "submit" button is disabled. Filled all mandatory fields.

Did you also give it a title? It is not visible in the screenshot

> click on the measurement tool. Nothing happen.

That is not a measurement tool, it only shoes the dimensions of the selected area. We have had some feature requests to let the user choose the dimension using this tool, but that is not implemented.
